src/components/Checkbox/Checkbox.tsx

```diff
@@ -9,6 +9,9 @@ import { st, classes } from './Checkbox.st.css';
 interface OnChangeEvent extends React.ChangeEvent<HTMLInputElement> {
   checked: boolean;
 }
+
+type Theme = 'box';
```

Comment:
We use this convention:
```suggestion
export enum CheckboxTheme {
  Default = 'default',
  Box = 'box',
};
```
Then we use it in the rest of the files (tests, storybook, etc.)